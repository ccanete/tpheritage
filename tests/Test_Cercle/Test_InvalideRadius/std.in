C test 12 34
EXIT