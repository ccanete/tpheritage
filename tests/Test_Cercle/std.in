C test 12 34 10
EXIT