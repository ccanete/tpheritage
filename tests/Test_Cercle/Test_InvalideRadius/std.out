ERR
# Invalide radius.

# The program is being closed.